### ISSUE TITLE
[Bug] the scale in extensionMark not working in LinearProgress.

### ISSUE THREAD

**Issue by youngwinds** [VisActor/VChart#582]
### Version

1.2.0

### Link to Minimal Reproduction

https://visactor.bytedance.net/vchart/demo/progress/linear-progress-with-text?keyword=linearProgress

### Steps to Reproduce

```
const spec = {
  type: 'linearProgress',
  data: [
    {
      id: 'id0',
      values: [
        {
          type: 'Name',
          value: 0.5,
          progress: 0.1,
          text: '79.5%'
        }
      ]
    }
  ],
  direction: 'horizontal',
  xField: 'value',
  yField: 'type',
  extensionMark: [
    {
      type: 'text',
      dataId: 'id0',
      visible: true,
      style: {
        x: 0,
        y: 0,
        stroke: 'red',
        text: {
          "type": "threshold",
          "field": "progress",
          "domain": [
            0.3333,
            0.6666
          ],
          "range": [
            "abc",
            "'e'",
            "'ab'"
          ]
        },
        textAlign: 'start',
        textBaseline: 'middle',
        lineWidth: 1
      }
    }
  ]
};
const vchart = new VChart(spec, { dom: CONTAINER_ID });
vchart.renderAsync();

// Just for the convenience of console debugging, DO NOT COPY!
window['vchart'] = vchart;
```

### Current Behavior

<img width="1518" alt="image" src="https://github.com/VisActor/VChart/assets/49136199/492683db-b6ee-4ed7-b883-048c28aa4415">

### Expected Behavior

text mark should render "abc"

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by xuefei1313** [comment]
will fix by upgrading vscale to 0.16.1

**Comment by kkxxkk2019** [comment]
https://github.com/VisActor/VUtil/pull/108
